projects/angular/src/popover/tooltip/_variables.tooltip.scss

```diff
@@ -10,7 +10,8 @@
 // Usage: ../popover/tooltip/_tooltips.clarity.scss
 $clr-tooltip-arrow-height: #{tokens.$cds-global-space-5} !default;
 $clr-tooltip-arrow-width: #{tokens.$cds-global-space-5} !default;
-$clr-tooltip-adjusted-margin: #{tokens.$cds-global-space-9} !default;
+// arrow width/height (12px) + distance (8px)
+$clr-tooltip-adjusted-margin: calc(tokens.$cds-global-space-6 + tokens.$cds-global-space-5) !default;
```

Comment:
The arrow width/height variables above are space-5 (8px), but the comment says 12px.

Reply:
The actual arrow is 12px and the distance from the trigger is 8px.
Basically the arrow creation i using both width and height to create a triangle, with one side 16px, and nudged 4px into the tooltip.

Please test it. It's dificult to explain in writing.

![image](https://github.com/user-attachments/assets/3c3cc9dd-c2bf-4496-ae3d-0001dda99e4c)